soweego/linker/classify.py

```diff
@@ -94,14 +94,7 @@ def execute(catalog, entity, model, name_rule, threshold, dir_io):
 
         _add_missing_feature_columns(classifier, feature_vectors)
 
-        if isinstance(classifier, rl.NaiveBayesClassifier):
-            predictions = classifier.prob(feature_vectors)
-        elif isinstance(classifier, rl.SVMClassifier):
-            predictions = classifier.predict(feature_vectors)
-        else:
-            err_msg = f'Unsupported classifier: {classifier}. It should be one of {set(constants.CLASSIFIERS)}'
-            LOGGER.critical(err_msg)
-            raise ValueError(err_msg)
+        predictions = classifier.prob(feature_vectors)
```

Comment:
```suggestion
        predictions = classifier.predict(feature_vectors) if isinstance(classifier, rl.SVMClassifier) else classifier.prob(feature_vectors)
```

`LinearSVC` doesn't emit probability scores: I'm still keeping support for it.